src/Sylius/Bundle/AdminBundle/templates/product/show/content/header/title_block/title.html.twig

```diff
@@ -5,3 +5,6 @@
         <h1 class="page-title">{{ product.name }}</h1>
     </div>
 </div>
+<div class="col-12 col-md-auto ms-auto d-print-none">
+    {% hook 'actions' %}
+</div>
```

Comment:
Pretty sure this is unnecessary, but if so it's in the wrong place, it's a part of the title block, but not of the title itself.